package.json

```diff
@@ -22,36 +21,39 @@
     "dist": "tsc && electron-builder --macos --linux --windows --publish never"
   },
   "dependencies": {
-    "electron-context-menu": "^0.12.0",
-    "electron-debug": "^2.1.0",
+    "electron-context-menu": "^0.14.0",
+    "electron-debug": "^3.0.1",
     "electron-dl": "^1.14.0",
     "electron-log": "^3.0.5",
-    "electron-settings": "^3.2.0",
-    "electron-store": "3.1.0",
+    "electron-store": "4.0.0",
     "electron-updater": "^4.0.6",
-    "electron-util": "^0.11.0",
-    "element-ready": "^3.1.0"
+    "electron-util": "^0.12.1",
+    "element-ready": "^4.1.0"
   },
   "devDependencies": {
-    "@sindresorhus/tsconfig": "^0.3.0",
-    "@types/electron-settings": "^3.1.1",
-    "@types/electron-store": "^1.3.1",
-    "@types/semver": "^5.5.0",
-    "@typescript-eslint/eslint-plugin": "^1.6.0",
+    "@sindresorhus/tsconfig": "^0.4.0",
+    "@types/semver": "^6.0.1",
+    "@typescript-eslint/eslint-plugin": "^2.0.0",
+    "@typescript-eslint/parser": "^2.0.0",
     "electron": "3.0.4",
-    "electron-builder": "^20.38.5",
-    "eslint-config-prettier": "^4.1.0",
-    "eslint-config-xo-typescript": "^0.9.0",
-    "husky": "^0.14.3",
-    "lint-staged": "^8.1.5",
+    "electron-builder": "^21.2.0",
+    "eslint-config-prettier": "^6.1.0",
+    "eslint-config-xo-typescript": "^0.17.0",
+    "husky": "^3.0.4",
+    "lint-staged": "^9.2.5",
     "npm-run-all": "^4.1.5",
     "prettier": "^1.16.4",
-    "stylelint": "^9.10.1",
+    "stylelint": "^10.1.0",
     "stylelint-config-prettier": "^5.0.0",
-    "stylelint-config-xo": "^0.13.0",
-    "typescript": "^3.3.4000",
+    "stylelint-config-xo": "^0.15.0",
+    "typescript": "~3.5.3",
```

Comment:
Why did this change from `^` to `~`?

Reply:
It was trying to pull in TypeScript 3.6.x which one of the packages is not yet compatible with.